internal/conf/configuration.go

```diff
@@ -414,6 +416,59 @@ type WebhookConfig struct {
 	Events     []string `json:"events"`
 }
 
+// Moving away from the existing HookConfig so we can get a fresh start.
+type HookConfiguration struct {
+	MFAVerificationAttempt ExtensibilityPointConfiguration `json:"mfa_verification_attempt" split_words:"true"`
+}
+
+type ExtensibilityPointConfiguration struct {
+	URI      string `json:"uri"`
+	Enabled  bool   `json:"enabled"`
+	HookName string `json:"hook_name"`
+}
+
+func (h *HookConfiguration) Validate() error {
+	points := []ExtensibilityPointConfiguration{
+		h.MFAVerificationAttempt,
+	}
+	for _, point := range points {
+		if err := point.ValidateAndPopulateExtensibilityPoint(); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func (e *ExtensibilityPointConfiguration) ValidateAndPopulateExtensibilityPoint() error {
+	if e.URI != "" {
+		regExp := `^[a-zA-Z_][a-zA-Z0-9_]{0,62}$`
+		re, err := regexp.Compile(regExp)
+		if err != nil {
+			return err
+		}
```

Comment:
Regexp compilation is slow. Please move this to the toplevel of the file and use `MustCompile`. The regex is specified by the programmer, it's not dynamic so an error is not possible.